backend/src/controllers/user.controller.ts

```diff
@@ -7,6 +7,9 @@ import {
   SigninSchema,
 } from "@shishuranjan/backend-common/dist/validations";
 import { generateToken } from "../utils/generateToken";
+import bcrypt from 'bcryptjs'
+import { log } from "console";
+import { get } from "http";
```

Comment:
What's up with this sir? Remove these.

Reply:
sorry i forgot to remove unused imports , i will do it on the fly

Reply:
do it asap @pandeyashu08 Its looks good for merging !!!